src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 17 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

PowerShellGet - Script tests (Admin).Should install a script correctly to the required location with AllUsers scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1: line 234
234:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```
minishell for native executables.Streams from minishell.gets the error stream from minishell
```powershell
Expected strings to be the same, but they were different.
Expected length: 49
Actual length:   16
Strings differ at index 0.
Expected: 'Microsoft....'
But was:  'RuntimeException'
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 30
30:             $output.FullyQualifiedErrorId | Should -Be 'Microsoft.PowerShell.Commands.WriteErrorException'
```
PowerShellGet - Script tests.Should install a script correctly to the required location with default CurrentUser scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1: line 207
207:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```
Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```
ForEach-Object -Parallel Basic Tests.Verifies non-terminating error streaming
```powershell
Expected strings to be the same, but they were different.
Expected length: 6
Actual length:   114
Strings differ at index 0.
Expected: 'Error!'
But was:  'Objects ca...'
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 58
58:         $actualError.ToString() | Should -BeExactly 'Error!'
```